3.8/arangosearch-analyzers.md

```diff
@@ -489,8 +489,16 @@ be run on DB-Servers in case of a cluster deployment. User-defined functions
 are not permitted.
 
 The input data is provided to the query via a bind parameter `@param`.
-It is always a string. The query result should be a string, `null` or
-an array of strings (and `null`s).
+It is always a string. The query result will be converted according to
+`returnType` parameter value. 
```

Comment:
Does the implementation literally convert the AQL result to _returnType_?
For example, if I `RETURN true` but set `returnType: "string"`, will it end up as string `"true"`?
And if I `RETURN "true"`  and set `returnType: "bool"`, will it end up as boolean `true`?

Reply:
Exactly